.dotnet.azure/sdk/openai/tools/TestFramework/src/OpenAI.TestFramework.csproj

```diff
@@ -18,7 +18,9 @@
     <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Moq" />
     <PackageReference Include="System.ValueTuple" />
-    <PackageReference Include="System.ClientModel" VersionOverride="1.1.0-beta.5" />
+    <PackageReference Include="System.ClientModel" VersionOverride="1.1.0" />
+
+    <ExcludeFromProjectReferenceToConversion Include="System.ClientModel" />
```

Comment:
Do we need this anymore?

Reply:
We need to keep this change in for the Azure C# SDK repo since the Azure C# SDK repo has build step that tries to replace Nuget package references with project references. Otherwise we will continue to have build breakages due to the dependencies between System.ClientModel (in Azure repo which needs to build and publish first),  OpenAI (in separate repo which consumes SCM via Nuget package), and Azure OpenAI (in Azure repo which consumes OpenAI and SCM via Nuget packages).

Keeping here also makes it easier to move code between Jose's repo and the Azure C# SDK repo.

Reply:
Cool, we can rip all of those out later once the upgrades go through - I have an issue tracking that.